### PR TITLE
Create dropdown component (#131)

### PR DESCRIPTION
**Pull request by mmazanec22**
This component raised a few odd little problems:

- Needed to define theme image path and font path relative to USWDS files using them
- Due to a [bug](https://github.com/uswds/uswds/issues/3117), uswds-theme-components was not included by default, so those are now included in index.scss
- Defining the image path broke the hero image path, because it used image path by default but was used in a different place-- upgrade to USWDS 2.2.1 fixed this by allowing us to set hero-image-path

### CODE REVIEW

File: src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+				}`}
+				name={selectId}
+				id={selectId}
+				onChange={onChange}
+				disabled={disabled}
+				aria-describedby={error ? errorMessageId : undefined}
+			>
+				<option value={undefined}>{unselectedOptionText}</option>
+				{options.map(option => (
+					<option value={option.value} key={option.value}>
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: Nit: Do we want to use `option.value` as the key? I've generally seen the index in the array be used for the key, but am not 100% on this.

File: src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	label,
+	unselectedOptionText,
+	onChange,
+	id,
+	success,
+	disabled,
+	error,
+	errorMessage,
+}: DropdownProps) {
+	const selectId = id || label.split(' ').join('-');
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: Is there a reason for constructing the `selectId` as so vs making `id` a required prop value?

File: src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: I think React yells at you if you use the index.  Option.value might not be the best either, though, considering the possibility for duplication.  Maybe a combination?

Comment (cailyncodes) [reply src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: I like the idea of a combination!

File: src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx (hunk 2 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: Out of sheer laziness I like defaults, but can make this required if that seems more sensisble

Comment (cailyncodes) [reply src/Hedwig/ClientApp/src/components/Dropdown/Dropdown.tsx]: I'm not opposed to having a sensible default. I was just curious if there was motivating reason. I think you can leave this as is 😃

**Pull request merged**